build/phase.go

```diff
@@ -161,6 +161,15 @@ func (p *Phase) Run(ctx context.Context) error {
 		return errors.Wrapf(err, "failed to copy files to '%s' container", p.name)
 	}
 
+	p.logger.Debugf("Running container: %v", p.ctr.ID)
+	if ctrConf := p.ctrConf; ctrConf != nil {
+		p.logger.Debugf("  Cmd: %v", ctrConf.Cmd)
+		p.logger.Debugf("  Env: %v", ctrConf.Env)
```

Comment:
I think including the `Env` here might be misleading. When users pass environment variables to `pack build` with the `--env` flag they actually get written to disk at `/platform/env` as described in the spec https://github.com/buildpack/spec/blob/master/buildpack.md#provided-by-the-platform.

I think seeing an empty Env array after providing an `--env` flag is likely to confuse users even when everything is working as expected.

Reply:
@ekcasey While I agree that it could be confusing, wouldn't this be useful information?

Alternatively, could this information be logged in lifecycle instead of `pack`? That way we could read the actual env vars, which should include the image-configured vars PLUS the `--env` vars (correct me if I'm wrong @ekcasey). One issue with that might be that there's no way (currently, I believe) for `pack` to tell lifecycle when to be verbose, so if it were logged in lifecycle, it might always show up.